kolibri/logger.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import absolute_import
+
+import logging
+
+from colorlog import ColoredFormatter
+
+# TODO: Logging decision.
+# The main Kolibri logger always logs to console and does not propagate.
```

Comment:
I agree. We shouldn't stop propagation. We should look for a holistic approach that unifies Kolibri's logging with every other library's logging. Then we can use the Django logging configuration settings as the final output filtering instead. I don't even think we should have the below, I was just so in love with the beauty of the logging colors :) But serious logging will allow the dev / deployment to handle how logging should be collected and displays --- with our defaults being very sane of course. A log file sounds like a good idea.